ShareX.UploadersLib/FileUploaders/Lithiio.cs

```diff
@@ -0,0 +1,105 @@
+﻿#region License Information (GPL v3)
+
+/*
+    ShareX - A program that allows you to take screenshots and share any file type
+    Copyright (c) 2007-2016 ShareX Team
+
+    This program is free software; you can redistribute it and/or
+    modify it under the terms of the GNU General Public License
+    as published by the Free Software Foundation; either version 2
+    of the License, or (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program; if not, write to the Free Software
+    Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+    Optionally you can also view the license at <http://www.gnu.org/licenses/>.
+*/
+
+#endregion License Information (GPL v3)
+
+// Copied Lambda.cs FileUploader and modified it a little to work with https://lithi.io/.
+// To-do: add the other domains to the dropdown menu.
+// Credits: https://github.com/mstojcevich
+// Minor changes: https://github.com/lithium720
+
+using Newtonsoft.Json;
+using System.Collections.Generic;
+using System.IO;
+using System.Windows.Forms;
+
+namespace ShareX.UploadersLib.FileUploaders
+{
+    public class LithiioFileUploaderService : FileUploaderService
+    {
+        public override FileDestination EnumValue { get; } = FileDestination.Lithiio;
+
+        public override bool CheckConfig(UploadersConfig config)
+        {
+            return config.LithiioSettings != null && !string.IsNullOrEmpty(config.LithiioSettings.UserAPIKey);
+        }
+
+        public override GenericUploader CreateUploader(UploadersConfig config, TaskReferenceHelper taskInfo)
+        {
+            return new Lithiio(config.LithiioSettings);
+        }
+
+        public override TabPage GetUploadersConfigTabPage(UploadersConfigForm form) => form.tpLithiio;
+    }
+
+    public sealed class Lithiio : FileUploader
+    {
+        public LithiioSettings Config { get; private set; }
+
+        public Lithiio(LithiioSettings config)
+        {
+            Config = config;
+        }
+
+        private const string uploadUrl = "http://api.lithi.io/upload.php";
+
+        public static string[] UploadURLs = new string[] { "https://lithi.io/" };
```

Comment:
I left this drop down menu here because I intend to add my other domains in the future.